sys/net/link_layer/l2util/l2util.c

```diff
@@ -238,5 +238,25 @@ int l2util_ndp_addr_len_from_l2ao(int dev_type,
     return -ENOTSUP;
 }
 
+int l2util_ipv6_group_to_l2_group(int dev_type,
+                                  const ipv6_addr_t *ipv6_group,
+                                  uint8_t *l2_group)
+{
+    switch (dev_type) {
+#if defined(MODULE_NETDEV_ETH) || defined(MODULE_ESP_NOW)
```

Comment:
does esp_now also support multicast?

Reply:
If it does it doesn't use it

https://github.com/RIOT-OS/RIOT/blob/ed64c603abeb2668b9a112bcc508f28d93b4c744/cpu/esp_common/esp-now/esp_now_gnrc.c#L53-L56

(and since the IPv6-over-ESP-now is proprietary to RIOT, I think that is alright currently)

Reply:
so we can drop the `|| defined(MODULE_ESP_NOW)`?

Reply:
Ah, yes. Somehow thought it wass `ESP_WIFI`.